packages/js-json-provider/src/lib/evaluation-engine.ts

```diff
@@ -1,20 +1,15 @@
-import {
-  FlagValueType,
-  Context,
-  ProviderEvaluation,
-  FlagNotFoundError,
-  TypeMismatchError,
-} from '@openfeature/openfeature-js';
 import { OpenFeatureFeatureFlags } from './flag';
 import { camelCase } from 'change-case';
+import { EvaluationContext, ResolutionDetails } from '@openfeature/nodejs-sdk';
+import { FlagNotFoundError, TypeMismatchError } from '@openfeature/extra';
 
 export class EvaluationEngine {
   evaluate<T>(
     flags: OpenFeatureFeatureFlags,
     flagKey: string,
-    returnType: FlagValueType,
-    context: Context
-  ): ProviderEvaluation<T> {
+    returnType: 'boolean' | 'string' | 'number' | 'object',
```

Comment:
We export a `FlagType` from the SDK that's an alias of this union type. I actually wonder if `FlagValueType` is a better name. What do you think? Should we update it in the SDK?

Reply:
https://github.com/open-feature/node-sdk/pull/18